<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pag_Principal</class>
 <widget class="QMainWindow" name="Pag_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_desc">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>203</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_titulo">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>234</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_fecha">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>265</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_buscar_texto">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_publicar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Publicar</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>611</width>
      <height>384</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>592</width>
       <height>465</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="label1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="label2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="label3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="label4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="label5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="label6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="label7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="label8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="label9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="label10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="label11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPushButton" name="label12">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="label13">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="label14">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="8" column="0">
         <widget class="QPushButton" name="label1_9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPushButton" name="label1_12">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="label1_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="label1_11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="label1_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="label1_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="label1_10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="label1_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="label1_13">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="label1_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="label1_14">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="label1_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="label1_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="label1_1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>126</width>
      <height>155</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_igual">
       <property name="text">
        <string>Buscar igual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_empieza">
       <property name="text">
        <string>Buscar Empieza</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_no_contenido">
       <property name="text">
        <string>Buscar no contenido</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_buscar_texto">
       <property name="text">
        <string>Buscar Texto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_temina">
       <property name="text">
        <string>Buscar termina</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txt_Buscar_Users"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Buscar">
       <property name="text">
        <string>Buscar Usuario</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>256</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_ord_likes">
       <property name="text">
        <string>Likes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ord_fecha">
       <property name="text">
        <string>Fecha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ord_num_letra">
       <property name="text">
        <string>Letra</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
